src/z3c/jbot/patches.py

```diff
@@ -128,7 +128,7 @@ def get_and_bind(template, view=None, cls=None):
 
     def get_skin_obj(obj, view=None, cls=None):
         layer = utility.getLayer()
-        key = layer, obj
+        key = layer, obj.filename
```

Comment:
I never used it to override scripts, but it seems to me that `.filename` is not specific enough.
AFAICS `obj` is a `Data.fs` object, right?
What about:
```suggestion
        key = layer, obj.getPhysicalPath()
```
?

Reply:
BTW, thanks @djay for this work!